### PR TITLE
Adds is service change option extension field

### PR DESCRIPTION
**Pull request by cmonagle**

### CODE REVIEW

**Review by rsecob — APPROVED** [review]
lgtm

File: nodejs/gtfs-realtime.js [hunk 1]
```diff
@@ -77,6 +86,8 @@ $root.TransitAlertExtension = (function() {
         writer.uint32(/* id 1, wireType 0 =*/8).uint64(message.createdAt);
         if (message.sourceName != null && message.hasOwnProperty("sourceName"))
             writer.uint32(/* id 2, wireType 2 =*/18).string(message.sourceName);
+        if (message.isServiceChangeAlert != null && message.hasOwnProperty("isServiceChangeAlert"))
```
Comment (lvancraen) [inline nodejs/gtfs-realtime.js:89]: Any reason we dont do a strict comparison here? `!=` vs. `!==`

**Review by lvancraen — COMMENTED** [review]
Other than the question about strictness, lgtm. But Im also not familiar with this project. So not sure I have a huge input here other than commenting on style

File: nodejs/gtfs-realtime.js (hunk 1 above)
Comment (cmonagle) [reply nodejs/gtfs-realtime.js:89]: Sorry Louis, I should have specified, the JS code in this project is automatically generated. I'll add a comment for the one actual line I changed.

I'm guessing the generated code needs to capture all nullish values (ie undefined but not 0).

File: transit-extensions.proto [hunk 2]
```diff
@@ -14,6 +14,10 @@ message TransitAlertExtension {
 
     // Transit's source name for that alert's agency
     optional string source_name = 2;
+
+    // Flag for automatically generated service change alerts
+    optional bool is_service_change_alert = 3;
```
Comment (cmonagle) [inline transit-extensions.proto:19]: this is the only actual change

**Pull request merged**